src/components/SingleCard.jsx

```diff
@@ -1,15 +1,25 @@
-const SingleCard = (props) => {
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function SingleCard(props) {
```

Comment:
Destructured props make the code easier to read/follow `({ id,  key, image, name, body })` instead of `(props)`

Reply:
that'll definitely make it more readable, thanks